[Add] Add italian translation, change field class to edit-only for better-zip field

FrancoTampieri opened the pull request:
Hi following this discussion https://github.com/OCA/partner-contact/issues/33 I made a PR.

Commit:
[Add] Add italian translation, change field class to edit-only for better-zip field

Commit:
[Add] Add some italian translation

pedrobaeza commented:
Hi, Franco, thanks for the changes, but propagate your name as contributor in all files is excessive, having in account that you have only add a class tags in a XML file. This is also an old technique that must be deprecated as soon as possible, because it's very unfair about the proportionality each one contribute. GitHub now tell us what everyone has done.

FrancoTampieri commented:
Hi Pedro,

no problem, I had seen the lines with the contributors name and I have added myself, but did not know there were any particular policy. I remove my name and made the PR.
I've added IT translation on the PR

Regards

Commit:
[Fix] Removed my contributor lines

scigghia commented:
:+1:

pedrobaeza commented:
:+1: 

Thanks for the changes.

tafaRU commented on base_location_geonames_import/i18n/it.po:
```diff
@@ -0,0 +1,79 @@
+#: code:addons/base_location_geonames_import/wizard/geonames_import.py:90
+#, python-format
+msgid "Error:"
+msgstr "Errore :"
```
Please, remove whitespace before ':'. Thank you.

FrancoTampieri replied:
Hi Tafaru I've fixed the issue in ti.po, can you try it and give me feedback?

Regards

Commit:
[Fix] Fix it.po translation

tafaRU commented:
:+1:

OpenCode commented:
:+1:

tafaRU commented:
~~I think Travis build failed due to https://github.com/OCA/partner-contact/pull/31~~
Travis fails after running test_pylint.

FrancoTampieri commented:
@tafaRU So we have to solve that PR to merge this?

FrancoTampieri commented:
The problem in thavis is a pylint failure on module passport and base_parter_merge

`************* Module partner-contact.passport.tests.test_passport
passport/tests/test_passport.py:36: [W0102(dangerous-default-value), Base_Test_passport.setUp] Dangerous default value {} as argument`

`************* Module partner-contact.base_partner_merge.base_partner_merge
base_partner_merge/base_partner_merge.py:29: [W0102(dangerous-default-value), html_entity_decode_char] Dangerous default value htmlentitydefs.entitydefs ({}) as argument`

Can I help to fix this issue?

tafaRU commented:
@drdran you had received more than 3 approves so I think your PR can be merged ;-)
My comment is just meant to point out the reason the Travis build was failing.

FrancoTampieri commented:
@tafaRU thanks for clearing up the problem:) anyway is if you need a hand to solve the problem with travis renew my call for help

tafaRU commented:
Thank you @drdran
I'm in favour to merge the PR as is, does anybody have any objections?

pedrobaeza commented:
OK, we proceed
